page.css

```diff
@@ -0,0 +1,98 @@
+
+html,
+body {
+    background-color: #FFE1E9;
+    font-family: sans-serif;
+}
+
+h1 {
+    font-size: 60px;
+}
+
+@media only screen and (min-width: 375px) {
```

Comment:
what if we just remove this media query? can we start from 320 by default?

Reply:
sure